presto-docs/src/main/sphinx/release/release-0.207.rst

```diff
@@ -0,0 +1,65 @@
+=============
+Release 0.207
+=============
+
+General Changes
+---------------
+
+* Fix a planning issue for queries where correlated references were used in ``VALUES``.
+* Remove support for legacy ``JOIN ... USING`` behavior.
+* Change behavior for unnesting an array of ``row`` type to produce multiple columns.
+* Deprecate ``reorder_joins`` session property and replace with ``join_reordering_strategy``
+  (along with the corresponding configuration property). Setting this new property to ``NONE``
+  maintains the order of the joins as written and is equivalent to ``reorder-joins=false``.
+  ``ELIMINATE_CROSS_JOINS`` will eliminate any unnecessary cross joins from the plan and is
+  equivalent to ``reorder-joins=true``. ``COST_BASED`` will use the new cost-based optimizer to
+  select the best join order. Setting the ``reorder_joins`` session property overrides the
+  ``join_reordering_strategy`` session property and ``optimizer.join-reordering-strategy``
+  configuration property.
+* Deprecate ``distributed_joins`` session property and replace with ``join_distribution_type``
+  (along with the corresponding configuration property). Setting this new property to
+  ``REPARTITIONED`` turns on hash partitioned joins and is equivalent to
+  ``distributed-joins-enabled=true``. ``BROADCAST`` sets the join strategy to broadcast and is
+  equivalent to ``distributed-joins-enabled=false``. ``AUTOMATIC`` will determine the join strategy based on the
+  estimated cost of each join.  If no statistics are available, a hash-partitioning strategy is used.
+  Setting the ``join_distribution_type`` session property overrides the ``join_distribution_type``
+  session property and ``join-distribution-type`` configuration property.
+* Add optimizer to reorder inner joins and choose the join type based on cost when statistics are available.
+* Add support for column properties.
+* Add ``optimizer.max-reordered-joins`` configuration property to set the maximum number of joins that
+  can be reordered at once using cost-based join reordering.
+* Add support for ``char`` type to :func:`approx_distinct`.
+
+Security Changes
+----------------
+
+* Change server to fail on startup when configuration for file based access controllers is invalid.
```

Comment:
See previous comment. Is this referring to connectors or a specific system access control? For connectors, split out for each affected connector. For system accsss control, mention which one.

Reply:
Atop, hive, raptor and mysql can all use the fileaccesscontrol if they choose. Should I create a heading for all of them?